zap/src/main/java/org/parosproxy/paros/core/scanner/Variant.java

```diff
@@ -24,6 +24,7 @@
 // ZAP: 2019/06/01 Normalise line endings.
 // ZAP: 2019/06/05 Normalise format/style.
 // ZAP: 2020/08/27 Added default methods for modifying the Sites tree
+// ZAP: 2020/05/06 Add method to get a short name of the variant
```

Comment:
These comments should be ordered by date.

Reply:
The year should be 2021.